src/clarity.rs

```diff
@@ -692,6 +694,7 @@ fn consume_arg(
     }
 }
 
+/// This function uses Clarity1 to parse the boot code.
```

Comment:
Is this going to use Clarity1 until some point in time? Or will always do so?

Reply:
I'm not sure, I'd like feedback on this as well. I updated the description to reflect this.

Reply:
Yes -- this should always use Clarity1, unless there's some sort of catastrophic bug. The "boot contracts" (or maybe more aptly named genesis contracts) shouldn't be modified in any future network upgrades, so they'd remain Clarity1.